docs/accordproject-concepts.md

```diff
@@ -93,13 +93,32 @@ _More information about the Accord Project markup can be found in the [CiceroMar
 
 ![Template Model](assets/020/template_model.png)
 
-Compared to a standard document template (e.g., in Word or pdf), Accord Project templates associate a _model_ to the natural language text. The model lets you categorize variables (is it a number, a monetary amount, a date, a reference to a business or organization, etc.). This model is crucial in that it allows the computer to make sense of the information contained in your template.
+Compared to a standard document template (e.g., in Word or pdf), Accord Project templates associate a _model_ to the natural language text. The model acts as a bridge between the text and logic; it gives the users an overview of the components, as well as the traits of different components.
 
-The following screenshot shows the model for the acceptance of delivery clause above.
+The model lets you categorize variables (is it a number, a monetary amount, a date, a reference to a business or organization, etc.). This model is crucial, because it allows the computer to make sense of the information contained in your template.
 
-![Example Model](/img/model-vscode.png)
+The following shows the model for the **Acceptance of Delivery** clause above.
 
-Thanks to that model, the computer now knows that the `shipper` variable (`"Party A"` in the example above) contains a reference to an `Organization`, that variable `receiver` (`"Party B"` in the example above) also contains a reference to an Organization, that variable `deliverable` (`"Widgets"` in the example above) can contain any text description, etc.
+```ergo
+/* The template model */
+asset AcceptanceOfDeliveryClause extends AccordClause {
+  /** the shipper of the goods*/
+  --> Organization shipper
+  /** the receiver of the goods */
+  --> Organization receiver
```

Comment:
Formatting could be synced up with the rest of this code snippet by having a space in between each of these.